client/client-multi/client-c/src/kaa/platform/kaa_client_properties.h

```diff
@@ -24,8 +24,16 @@
 #ifndef POSIX_KAA_CLIENT_PROPERIES_H_
 #define POSIX_KAA_CLIENT_PROPERIES_H_
 
-typedef struct {
-        unsigned long max_update_time;
-} kaa_client_props_t;
+#include <kaa_error.h>
+
+typedef struct kaa_client_props_t kaa_client_props_t;
+
+kaa_error_t kaa_client_props_set(const kaa_client_props_t *self);
+
+kaa_client_props_t* kaa_client_props_get(void);
+
+kaa_error_t kaa_set_working_directory(const char* working_directory);
+
+char* kaa_get_working_directory(void);
```

Comment:
Shouldn't this be `const char*`? Is it intended that user can change the string returned by this function?